Networking/Networking/Remote/MetaDataRemote.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+
+/// Interface for remote requests to Product and Orders meta data.
+///
+public protocol MetaDataRemoteProtocol {
+    func updateMetaData(for siteID: Int64, for parentID: Int64, type: MetaDataType, metadata: [[String: Any]]) async throws -> [MetaData]
+}
+
+public final class MetaDataRemote: Remote, MetaDataRemoteProtocol {
+
+    // MARK: - MetaData
+
+    /// Updates metadata for a specific item.
+    ///
+    /// - Parameters:
+    ///     - site: Site for which we'll update the metadata.
+    ///     - request: The request containing metadata to be updated.
+    ///     - type: The type of item (order or product) for which we'll update the metadata.
```

Comment:
We should update the parameters to reflect the method. For example, I don't see `parentID` and `metadata`.

Reply:
Good catch, I updated it here 078f3beb43f517fdcbf266095c2a294ae54c47e6